ds-intermediate-python/syllabus.md

```diff
@@ -0,0 +1,29 @@
+Setting up Environment For Collaborative Code Development
+* Introduction to Our Software Project
+* Virtual Environments For Software Development
+* Integrated Software Development Environments
+* Collaborative Software Development Using Git and gitHub
+* Python Code Style Conventions
+* Verifying Code Style Using Linters
+
+Ensuring Correctness of Software at Scale
+* Automatically Testing Software
+* Scaling Up Unit Testing
+* Continuous Integration for Automated Testing
+* Diagnosing Issues and Improving Robustness
+
+Software Development as a Process
+* Software Requirements
+* Software Architecture and Design
+* Programming Paradigms
+* Functional Programming Intuition
+* Object Oriented Programming Intuition
+
+Collaborative Software Development for Reuse
+* Developing Software in a Team: Code Review
+* Preparing Software for Reuse and Release
+* Packaging Code for Release and Distribution
+
+Managing and Improving Software over Its Lifetime
```

Comment:
Here we skip https://carpentries-incubator.github.io/python-intermediate-development/52-assessing-software-suitability-improvement/index.html